runtime/src/asset.rs

```diff
@@ -2332,6 +2265,12 @@ mod tests {
                 true
             ));
 
+            assert_eq!(
```

Comment:
`assert_eq` panics. We could use `ensure` instead.

Reply:
These are inside tests. These panics will mark the test case as failed and that's what we want.